src/containers/TaskRuns/TaskRuns.js

```diff
@@ -49,93 +57,76 @@ export /* istanbul ignore next */ class TaskRunsContainer extends Component {
   }
 
   state = {
-    notification: null,
-    loading: true,
     selectedStepId: null,
     selectedTaskId: null,
-    taskRuns: []
+    loading: true
   };
 
-  async componentDidMount() {
-    try {
-      const { match } = this.props;
-      const { taskName } = match.params;
-      await this.loadTaskRuns(taskName);
-    } catch (error) {
-      let message = error;
-      if (error.response) {
-        message = error.response.status === 404 ? 'Not Found' : 'Error';
-      }
-      this.setState({
-        notification: { kind: 'error', message },
-        loading: false
-      });
-    }
+  componentDidMount() {
+    const { match } = this.props;
+    const { taskName } = match.params;
+    this.props
```

Comment:
`Promise.all` to fetch task and task runs in parallel, similar to PipelineRun container